src/pages/candidate-details/candidate-details.ts

```diff
@@ -107,4 +106,8 @@ export class CandidateDetailsPage extends BasePageComponent implements OnInit, O
   handleDoneButtonClick(): void {
     this.navController.pop();
   }
+  public specialNeedsIsPopulated(specialNeeds: string | string[]): boolean {
```

Comment:
If a string for special needs was passed instead of an array of strings it would return false for isArray.

Reply:
The only time it will be a string is when, the specialneeds is not populated, the selector will set that as a string 'None'. If specialneeds is a string from source, it is spit anyway turning into a single item array